File: competitive_programming/codeforces/06_javascript/118a_string_task.js
function stringTask(text) {
  const vowels = "aeiouy";
  let result = "";
  for (const character of text.toLowerCase()) {
    if (!vowels.includes(character)) result += `.${character}`;
  }
  return result;
}

if (require.main === module) {
  if (stringTask("Codeforces") !== ".c.d.f.r.c.s") throw new Error("string task failed");
  if (stringTask("aBAcAba") !== ".b.c.b") throw new Error("string task failed");
  console.log("118A string task ok");
}

File: competitive_programming/codeforces/09_rust/118a_string_task.rs
fn string_task(text: &str) -> String {
    let vowels = "aeiouy";
    let mut result = String::new();
    for character in text.to_lowercase().chars() {
        if !vowels.contains(character) {
            result.push('.');
            result.push(character);
        }
    }
    result
}

fn main() {
    assert_eq!(string_task("Codeforces"), ".c.d.f.r.c.s");
    assert_eq!(string_task("aBAcAba"), ".b.c.b");
    println!("118A string task ok");
}
